tests/end-to-end/migration/migration-from-95.phpt

```diff
@@ -5,12 +5,19 @@ Configuration migration from PHPUnit 9.5 format works
 $_SERVER['argv'][] = '--do-not-cache-result';
 $_SERVER['argv'][] = '--migrate-configuration';
 
+$originalPHPUnitXMLFile = __DIR__ . '/_files/migration-from-95/phpunit-9.5.xml';
+$phpunitXMLFile = sys_get_temp_dir() . '/phpunit.xml';
+
 chdir(sys_get_temp_dir());
-copy(__DIR__ . '/_files/migration-from-95/phpunit-9.5.xml', 'phpunit.xml');
+copy($originalPHPUnitXMLFile, $phpunitXMLFile);
 
 require_once __DIR__ . '/../../bootstrap.php';
 
 (new PHPUnit\TextUI\Application)->run($_SERVER['argv']);
+
+assert(
+    file_get_contents($originalPHPUnitXMLFile) === file_get_contents($phpunitXMLFile)
+);
```

Comment:
disclaimer: I am no official phpunit developer and sebastian might have a different idea how such a test should look like.

----

I think you could do something like

```php
register_shutdown_function(function() use ($originalPHPUnitXMLFile, $phpunitXMLFile) {
    assert(
        file_get_contents($originalPHPUnitXMLFile) === file_get_contents($phpunitXMLFile)
    );
});

(new PHPUnit\TextUI\Application)->run($_SERVER['argv']);
```

instead

Reply:
This works, thanks for your suggestions!